tests/baselines/reference/outModuleConcatUmd.js.map

```diff
@@ -1,2 +1,2 @@
 //// [all.js.map]
-{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":";;;;;AACA,oBAAoB"}
\ No newline at end of file
+{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":";;;;;AACA,oBAAoB;;;;;;;;;;IAEpB;;;;SAAkB;IAAlB,YAAa,CAAC,CAAI;;;;;;;;;;;ICHlB,6BAA0B;IAC1B;;;;;;MAAuB,KAAC,GAAI;IAA5B,YAAa,CAAC,CAAc"}
```

Comment:
This is another case where I'd rather we not accept this file as it uses `outFile`.